nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -228,6 +228,9 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   - The `erlang_node_short_name`, `erlang_node_name`, `port` and `options` configuration parameters are gone, and have been replaced with an `environment` parameter.
     Use the appropriate [environment variables](https://hexdocs.pm/livebook/readme.html#environment-variables) inside `environment` to configure the service instead.
 
+- The `crystal` package has been updated to 1.11.x, which has some breaking changes.
```

Comment:
:+1:

Reply:
Maybe link to https://github.com/crystal-lang/crystal/blob/master/CHANGELOG.md#1110-2024-01-08 and say to look at it if you are affected?